pkg/shellquote/shellstring.go

```diff
@@ -1,4 +1,4 @@
-package logging
+package shellquote
```

Comment:
Because I put the background code in `pkg/proc` isntead of `pkg/client/cli/cliutil`, I had to move this out of `logging` to avoid an import loop.